assets/sass/_variables.scss

```diff
@@ -53,6 +53,9 @@ $facebook-border-color: #375685;
 $google-icon-bg-color: #fff;
 $google-label-bg-color: $button-bg-color;
 $google-border-color: darken($button-bg-color, 22%);
+$apple-icon-bg-color: #fff;
+$apple-label-bg-color: $button-bg-color;
+$apple-border-color: darken($button-bg-color, 22%);
```

Comment:
We should follow Apple's [design requirements](https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple/overview/buttons/) for web **Sign in with Apple** buttons.

Looking at the video - we might want to consider making the button background black to avoid adding a one-off border.

Reply:
we're certainly not strictly following apple guide line.. :(
looks like our current design align with `White with Outline` section.
@ZoeMonosson-okta ^

Reply:
Right - and I think it is ok that we're not matching 100%.

From what I've found, users who are accustomed to the Apple ecosystem tend to interact with familiar branding. As a frequent Sign-in/up With Apple user myself, I'd be concerned with a button that doesn't quite fit what I'm used to. Likewise, seeing a Sign-in with Facebook button that isn't blue would likely detour users.

Again, just my two cents. 😄

Reply:
Certainly a gap and we'd like to close as soon as we could.